rackhd_BIST/README.md

```diff
@@ -0,0 +1,31 @@
+## RackHD BIST 
+
+RackHD Built In Self Test can be used to check health status of RackHD environment and configuration.
+
+## rackhd_bist.py
+
+This script will run RackHD BIST and log necessary information.
+
+Follow the below instructions to use this tool:
+
+ 1. Configure following items in bist_user_config.json file:
+
+    + **sourceCodeRepo**: folder path that stores RackHD source code, if not configured it is "/var/renasar/"
+    + **logPath**: RackHD BIST log path, BIST logs will be stored in the path, if not configured it is "/var/log/rackhd/"
```

Comment:
The folder `/var/log/rackhd` need root permission to write data, but root permission is not required for BIST itself. So it's not worthy to grant root permission only for logging. Usually the practice is to write log in the same folder where the script stores.

My thought is default not output to the log file. Only if user specify a log path, then BIST will output some detail information into that file. Because I can image mostly user will not need a log file if the console output is clear enough.